test cases/common/220 fs module/subdir/meson.build

```diff
@@ -4,3 +4,49 @@ assert(fs.is_samepath(meson.project_source_root(), '..'), 'is_samepath not detec
 assert(fs.is_samepath(meson.project_build_root(), meson.current_build_dir() / '..'), 'is_samepath not detecting same directory')
 
 assert(fs.is_samepath(subdirfiles[0], 'subdirfile.txt'), 'is_samepath not detecting same directory when using File and str')
+
+# More relative_to to test subdir/builddir components
+
+python3 = find_program('python3')
+build_to_src = fs.relative_to(meson.current_source_dir(), meson.current_build_dir())
+src_to_build = fs.relative_to(meson.current_build_dir(), meson.current_source_dir())
+
+btgt = executable('btgt', 'btgt.c')
+ctgt = custom_target(
```

Comment:
You can use fs.copyfile(), it's easier and returns a CustomTarget as well.

Reply:
I wanted the `CustomTargetIndex`s for testing

Reply:
You can do [0] on a custom target that had only one output. But as you wish ;)

Reply:
Bleh, I will fix it.

Reply:
but let's not spin the CI for an hour.

Reply:
oh well